mixer/adapter/stackdriver/metric/bufferedClient.go

```diff
@@ -166,7 +166,8 @@ func (b *buffered) Send() {
 		if err != nil {
 			ets := handleError(err, timeSeries)
 			b.updateRetryBuffer(ets)
-			_ = b.l.Errorf("Stackdriver returned: %v\nGiven data: %v", err, timeSeries)
+			_ = b.l.Errorf("Stackdriver returned: %v\n", err)
```

Comment:
1. special case "out of order" and maintain a count of how many data points were rejected.
1a. I hope `err` is not a large error message.
2. It is ok to print other errors.

Reply:
It is really not reliable to figure out error type based on message, but it might be ok since it only affects the log output.

Number of points that are affected could be found from error message itself. An example server error message. An example error message: `Points must be written in order. One or more of the points specified had an older start time than the most recent point.: timeSeries[12,13,18,19,49,50]`

Reply:
Added more to the message about total time series sent in RPC. Is this what you are looking for?